scripts/seshat.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Starts seshat service-discovery agent service
+
+[Service]
+Environment="LOG4C_RCPATH=/rdklogger"
+ExecStart=/usr/bin/seshat -u tcp://192.168.101.3:7777
```

Comment:
Can the `tcp://192.168.101.3:7777` be `tcp://127.0.0.1:7777` since it's localhost's IP address.